Flattening Deeply Nested Data (KSQL) (#191)
confluentinc/kafka-tutorials: _includes/tutorials/flatten-nested-data/ksql/markup/dev/create-inputs.adoc

> Create the following stream that contains the representation of the order.

Reviewer: ```suggestion
Create the following stream with the representation of the orders.
```

> We will the `STRUCT` keyword to define the fields that contains nested data.

Reviewer: ```suggestion
We use the `STRUCT` keyword to define the fields that contain nested data.
```
